lib/ansible/config/module_defaults.yml

```diff
@@ -524,9 +524,9 @@ groupings:
   - azure
   azure_rm_loadbalancer_facts:
   - azure
-  azure_rm_managed_disk:
+  azure_rm_manageddisk:
```

Comment:
what 's  this file used for?

Reply:
Seems like the purpose of this file is to assign modules to groups.

And then user could use **module_defaults** like described here https://docs.ansible.com/ansible/latest/user_guide/playbooks_module_defaults.html
on **group/azure** for all azure modules, like this:

- hosts: localhost
  module_defaults:
    **group/azure**:
      location: eastus
  tasks:
    - azure_rm_xxxx:

But that doesn't seem to be very common feature, and I think it's not maintained very well.